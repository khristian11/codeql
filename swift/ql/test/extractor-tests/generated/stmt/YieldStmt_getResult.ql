import codeql.swift.elements
import TestUtils

from YieldStmt x, int index
where toBeTested(x)
select x, index, x.getResult(index)
