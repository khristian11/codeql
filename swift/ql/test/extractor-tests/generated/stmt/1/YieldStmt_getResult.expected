| statements.swift:77:7:77:7 | yield ... | 0 | file://:0:0:0:0 | &... |
| statements.swift:80:7:80:14 | yield ... | 0 | statements.swift:80:13:80:14 | &... |
